network-libp2p/src/discovery/peer_contacts.rs

```diff
@@ -433,14 +314,13 @@ impl PeerContactBook {
         self.peer_contacts.insert(peer_id, Arc::new(info));
     }
 
-    pub fn insert_filtered(
-        &mut self,
-        contact: SignedPeerContact,
-        protocols_filter: Protocols,
-        services_filter: Services,
-    ) {
+    pub fn insert_filtered(&mut self, contact: SignedPeerContact, services_filter: Services) {
         let info = PeerContactInfo::from(contact);
-        if info.matches(protocols_filter, services_filter) {
+        if info.matches(services_filter) {
+            log::debug!(
+                " Inserting peer_id: {} into my peer contacts, because is interesting to me, it provides: {:?}",
+                info.peer_id,info.services()
+            );
```

Comment:
```suggestion
            log::debug!(
                added_peer =  info.peer_id,
                services = info.services(),
                "Inserting into my peer contacts, because is interesting to me",
            );
```
Use the `tracing` way of logging